configs/docker.rc

```diff
@@ -22,6 +22,9 @@ supervisor__autostart_uwsgi = true
 supervisor__autostart_elasticsearch_server = false
 uwsgi__socket = 0.0.0.0:9000
 uwsgi__processes = 3
+# this means that the exposed nginx is secure (https) even
```

Comment:
Or we could migrate away from having a global nginx and an nginx container. But, not the scope of this PR.